packages/sdk/src/core/classes/erc-1155-lazymintable.ts

```diff
@@ -121,6 +121,7 @@ export class Erc1155LazyMintable implements DetectableFeature {
    *
    * @param metadatas - The metadata to include in the batch.
    * @param options - optional upload progress callback
+   * @twfeature ERC1155LazyMintable
```

Comment:
we don't want this one here, instead we want ppl to use the `erc1155.lazyMint` function directly from the `Erc1155` class.

consider this internal